tests/openapi-drift.test.ts

```
@@ -1,136 +1,172 @@
+      const swaggerPath = paths[normalizedPath];
+      if (!swaggerPath) continue;
+
+      const operation = swaggerPath[method.toLowerCase()];
+      if (!operation) continue;
+
+      // Convert Zod schema to JSON schema
+      let jsonSchema: any;
+      try {
+        jsonSchema = zodToJsonSchema(zodSchema);
+      } catch (e) {
+        errors.push(`[ERROR] ${routeKey}: Failed to convert Zod schema to JSON: ${(e as Error).message}`);
```

ESLint

[error] Replace ``[ERROR]·${routeKey}:·Failed·to·convert·Zod·schema·to·JSON:·${(e·as·Error).message}`` with `⏎··········`[ERROR]·${routeKey}:·Failed·to·convert·Zod·schema·to·JSON:·${(e·as·Error).message}`,⏎········`

(prettier/prettier)